Platforms/QemuQ35Pkg/.azurepipelines/NuGet.yml

```diff
@@ -0,0 +1,204 @@
+## @file
+# Azure Pipeline build file for publishing a NuGet package.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+parameters:
+- name: publish_q35_binaries
+  displayName: Publish QEMU Q35 Binaries
+  type: boolean
+  default: true
+- name: publish_q35sbsa_binaries
+  displayName: Publish QEMU Q35 SBSA Binaries
+  type: boolean
+  default: true
+
+name: $(Date:yyyyMMdd)$(Rev:.r) Mu Tiano Platforms NuGet Release
+
+jobs:
+  - job: Publish
+
+    strategy:
+      matrix:
+        ${{ if eq(parameters.publish_q35_binaries, true) }}:
+          QEMU Q35 Debug:
+            artifact_name: Binaries QemuQ35Pkg DEBUG
+            nuget_package_config_file: >-
+              https://raw.githubusercontent.com/microsoft/mu_tiano_platforms/main/Platforms/QemuQ35Pkg/NugetPublishing/DebugConfig.yaml
+            gcc_folder_name: Linux-x86
+            gcc_pipeline_name: Mu Tiano Platforms Q35 Plat CI GCC5
+            vs_folder_name: Windows-x86
+            vs_pipeline_name: Mu Tiano Platforms Q35 Plat CI VS
+
+          QEMU Q35 Release:
+            artifact_name: Binaries QemuQ35Pkg RELEASE
+            nuget_package_config_file: >-
+              https://raw.githubusercontent.com/microsoft/mu_tiano_platforms/main/Platforms/QemuQ35Pkg/NugetPublishing/ReleaseConfig.yaml
+            gcc_folder_name: Linux-x86
+            gcc_pipeline_name: Mu Tiano Platforms Q35 Plat CI GCC5
+            vs_folder_name: Windows-x86
+            vs_pipeline_name: Mu Tiano Platforms Q35 Plat CI VS
+
+        ${{ if eq(parameters.publish_q35sbsa_binaries, true) }}:
+          QEMU Q35 SBSA Debug:
+            artifact_name: Binaries QemuSbsaPkg DEBUG
+            nuget_package_config_file: >-
+              https://raw.githubusercontent.com/microsoft/mu_tiano_platforms/main/Platforms/QemuSbsaPkg/NugetPublishing/DebugConfig.yaml
+            gcc_folder_name: Linux-x86

```

Comment:
The x86 in the name for the Sbsa might also cause confusion due to it being a AARCH64(?) image.

Reply:
You're right. I will update it.

Reply:
Updated in PR commit https://github.com/microsoft/mu_tiano_platforms/pull/286/commits/3490a3f23a19fa84564d3e3fc9ca9bd40337f455